x-pack/plugins/lens/common/expressions/xy_chart/legend_config.ts

```diff
@@ -37,6 +37,14 @@ export interface LegendConfig {
    * Number of columns when legend is set inside chart
    */
   floatingColumns?: number;
+  /**
+   * Maximum number of lines per legend item
+   */
+  maxLines?: number;
+  /**
+   * Flag whether the legend is inside the chart
```

Comment:
I am not sure what you mean with this comment 🤔 ... isn't it the same as `Specifies whether the legend items should be truncated`?